client-app/src/examples/portfolio/detail/DetailPanelModel.js

```diff
@@ -1,17 +1,16 @@
-import {HoistModel, loadAllAsync, managed} from '@xh/hoist/core';
-import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {HoistModel, lookup, managed} from '@xh/hoist/core';
+import {observable, makeObservable} from '@xh/hoist/mobx';
 import {PanelModel} from '@xh/hoist/desktop/cmp/panel';
 import {OrdersPanelModel} from './OrdersPanelModel';
-import {ChartsPanelModel} from './ChartsPanelModel';
 import {PERSIST_DETAIL} from '../AppModel';
-
+import {PortfolioPanelModel} from '../PortfolioPanelModel';
 
 export class DetailPanelModel extends HoistModel {
 
-    @bindable positionId = null;
+    @observable positionId = null;
 
-    @managed ordersPanelModel = new OrdersPanelModel();
-    @managed chartsPanelModel = new ChartsPanelModel();
+    @lookup(PortfolioPanelModel) parentModel;
```

Comment:
Did we talk about not calling this "parentModel" anymore but rather naming it base don its class name? I actually don't mind parentModel, but it seems like not really the case anymore since DetailPanelModel doesn't get instantiated by the "parent" anymore.

Reply:
I don't think parentModel is now a verboten term. Let's stick with it to avoid churn